lib/gym/package_command_generator.rb

```diff
@@ -55,4 +81,53 @@ def dsym_path
       end
     end
   end
+
+  # Responsible for building the fully working xcodebuild command
+  class PackageCommandGeneratorV7
+    @config_content = %(<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
```

Comment:
I am not really sure how to deal with that. I haven't played with the different options.

I would need to port a project of ours to xcode7, because we for example need adhoc support and right now only appstore deployment would be supported.